### ISSUE TITLE
Detect, and fix, template tags broken over multiple lines

### ISSUE THREAD

**Issue by jarshwah** [adamchainz/djade#98]
### Description

Template tags [that are broken over multiple lines are rendered as text](https://code.djangoproject.com/ticket/8652) by DTL:

```
{% tagname ...
%}  <-- uh, oh! closed on a different line
```

It's a source of many innocuous bugs. Is this something that could be detected and fixed by djade?

**Comment by EmilStenstrom** [comment]
This is patched to work across many projects and some libraries (including django-components), so I suggest not applying this to avoid errors either way.

**Comment by adamchainz** [comment]
Yes, also there's ongoing work to add support for this to Django: https://forum.djangoproject.com/t/allow-newlines-inside-tags/36040

But a lint rule could be useful. I actually filed a duplicate the day after @jarshwah : #99.
